playgrounds/vite/src/WalletPicker.tsx

```diff
@@ -61,6 +62,7 @@ export const availableChainsByWallet = {
   [WalletOption.METAMASK]: EVMChains,
   [WalletOption.OKX_MOBILE]: EVMChains,
   [WalletOption.PHANTOM]: [Chain.Solana],
+  [WalletOption.POLKADOT_JS]: [Chain.Polkadot, Chain.Chainflip],
```

Comment:
This should be removed as well - I will do it now and change folder name and merge then

Reply:
done